#!/usr/bin/env python
PACKAGE = "prx_mapping"
# The following line should not be here for Groovy and above
#import roslib;roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("stop", 			bool_t,		0, "Stop capturing and save current world to world.pcd", 	False)
gen.add("first_rotating_axis",		int_t,    	1, "First rotating axis (0 = none, 1 = x, 2 = y, 3 = z)", 	1,  0, 3)
gen.add("first_rotating_axis_signal", 	double_t, 	0, "First rotating axis signal",    				1.0, -1.0, 1.0)
gen.add("second_rotating_axis",		int_t,    	0, "Second rotating axis (0 = none, 1 = x, 2 = y, 3 = z)", 	2,  0, 3)
gen.add("second_rotating_axis_signal", 	double_t, 	0, "Second rotating axis signal",    				1.0, -1.0, 1.0)
exit(gen.generate(PACKAGE, "prx_mapping", "prx_mapping"))
